Source/FullScreenPokemon.ts

```diff
@@ -4829,6 +4852,40 @@ module FullScreenPokemon {
                 callback);
         }
 
+        /* partyActivate functions
+         */
```

Comment:
There should be two blank lines before this. Also, the `*/` is one space to the right of where it should be.